### ISSUE TITLE
Exception when parsing valid response from GET appInfos/{app_info_id}/appInfoLocalizations

### ISSUE THREAD

**Issue by fiannaca** [microsoft/asconnect#3]
I am encountering an exception when running the following test script:

```py
client = asconnect.Client(key_id="****", key_contents="****", issuer_id="****")
app = client.app.get_from_bundle_id("****")

print("Connected to App Store Connect via API\n")
print("App: " + app.name + "\n")

info = client.app_info.get_app_info(app_id=app.identifier)

for item in info:
    print(item.identifier)

    # Exception occurs on the following line...
    localizations = client.app_info.get_localizations(app_info_id=item.identifier)

    for localization in localizations:
        print("\t" + localization.attributes.locale + ": " + localization.attributes.name)
```

Traceback of the exception:

```
Traceback (most recent call last):
  File "asconnect-test.py", line 20, in <module>
    for localization in localizations:
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/asconnect/app_info_client.py", line 63, in get_localizations
    yield from self.http_client.get(url=url, data_type=List[AppInfoLocalization])
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/asconnect/httpclient.py", line 126, in get
    deserialized_data = deserialize.deserialize(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 93, in deserialize
    return _deserialize(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 186, in _deserialize
    _deserialize_list(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 255, in _deserialize_list
    deserialized = _deserialize(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 175, in _deserialize
    _deserialize_dict(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 393, in _deserialize_dict
    deserialized_value = _deserialize(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 175, in _deserialize
    _deserialize_dict(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 393, in _deserialize_dict
    deserialized_value = _deserialize(
  File "/Users/alfianna/.local/share/virtualenvs/asconnect-test-xCg3XSFM/lib/python3.8/site-packages/deserialize/__init__.py", line 223, in _deserialize
    raise DeserializeException(
deserialize.exceptions.DeserializeException: Cannot deserialize '<class 'NoneType'>' to '<class 'str'>' for 'typing.List[asconnect.models.app_info.AppInfoLocalization][0].attributes.privacy_policy_text'
```

I'm running the following configuration:
* asconnect 1.0.2
* python 3.8
* MacOS 10.15.6

I think this exception is occurring because `AppInfoLocalization.Attributes.privacy_policy_text` is of type `str` instead of `Optional[str]` in [app_info.y:23](https://github.com/microsoft/asconnect/blob/main/asconnect/models/app_info.py#L23). Looking at the [API docs](https://developer.apple.com/documentation/appstoreconnectapi/appinfolocalization/attributes), I think all of the parameters of `AppInfoLocalization.Attributes` are optional.

I'll open a PR with a proposed fix momentarily.

**Comment by dalemyers** [comment]
Thanks for reporting this (and for fixing it!) Sorry that I didn't respond faster. Apparently I wasn't "watching" this repo.

**Comment by dalemyers** [comment]
The fix for this has been published in v2.0.1

**Comment by fiannaca** [comment]
No problem. Thanks!
